server/test/tempoutfitter-controllers.spec.es6

```diff
@@ -369,6 +369,23 @@ describe(`POST ${fileUploadUrl} accepts a file`, () => {
         done(err);
       });
   });
+
+  it('should accept a location-map file and return 201 created', done => {
+    request(server)
+      .post(fileUploadUrl)
+      .type('multipart/form-data')
+      .field('applicationId', applicationId)
+      .field('documentType', 'location-map')
+      .set('Accept', 'text/html')
+      .attach('file', './test/data/test.docx')
```

Comment:
should this be an acceptable file type for a location map?

Reply:
Good point, it doesn't seem like the middle layer is doing the filetype check, just the frontend, does that make sense or should there be an additional check server side?

Reply:
Yeah, this should probably check against the filetype that we want to supply.

Reply:
@hursey013 do you think you can update here?